Terminal.Gui/Core/PosDim.cs

```diff
@@ -375,6 +425,39 @@ internal virtual int Anchor (int width)
 			return 0;
 		}
 
+		internal class DimFunc : Dim {
```

Comment:
Same... needs conceptual documentation; a sentence or two that explains WHY someone would use this class.

Reply:
Done